code/game/machinery/computer/arcade.dm

```diff
@@ -206,9 +211,10 @@ GLOBAL_LIST_INIT(arcade_prize_pool, list(
 		return
 
 	if (!blocked && !gameover)
+		var/gamerSkill = usr.mind.get_skill_level(/datum/skill/gaming)
```

Comment:
you need to use the modifier, no?

Reply:
I think I'll continue to use your actual level for this, but prob() modifiers will use an actual modifier list